src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/custom.py

```diff
@@ -17,9 +17,11 @@ def start_shell(cmd, update=None, style=None):
     try:
         get_extension(INTERACTIVE_EXTENSION_NAME)
         if update:
+            logger.warning("Updating the Interactive extension to the latest available..")
```

Comment:
Isn't this done via option flag though?  So the user would already expect this.

Reply:
The wait time is large enough that I think it would be helpful to inform the user. In addition, I've received some feedback that for interactive, it is better to veer on the side of being more chatty/informative.